dist/complaint-tool-schema.json

```diff
@@ -427,12 +427,247 @@
                     "country": {
                       "type": "string",
                       "enum": [
-                        "US"
+                        "AFG",
```

Comment:
@lihanli @rianfowler do we have updated validation information for school address country?